lib/myfinance/resources/entity_collection.rb

```diff
@@ -0,0 +1,18 @@
+module Myfinance
+  module Resources
+    #
+    # A wrapper to Myfinance entities collection
+    #
+    class EntityCollection < Collection
+      attr_reader :name
+
+      private
+
+      def build_collection
+        Myfinance::Resources::Base.parsed_body(response).each do |attributes|
+          collection.push(Myfinance::Resources::Entity.new(attributes['entity']))
```

Comment:
Line is too long. [81/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.